README.md

```diff
@@ -58,12 +58,12 @@ The `examples` folder contains one framework example as of right now:
 
 - [CRBT](https://crbt.app/) (closed source)
 
-> Have an open-source bot of your own you'd like to feature? [Open an GitHub issue](https://github.com/CRBT-Team/Purplet/issues) and we may add it here!
+> **Note**: Have an open-source bot of your own you'd like to feature? [Open an GitHub issue](https://github.com/CRBT-Team/Purplet/issues) and we may add it here!
 
 ## Contributing
 
 See the [contributing guide](CONTRIBUTING.md) for information on how we accept contributions, as well as an overview of the codebase.
 
 ## License
 
-Purplet is licensed under the [Apache License 2.0](https://github.com/CRBT-Team/Purplet/blob/main/LICENSE).
+Purplet is licensed under the Apache License 2.0. View the full file [here](https://github.com/CRBT-Team/Purplet/blob/main/LICENSE).
```

Comment:
It's generally considered bad practice to hide an hyperlink in a generic statement like "here" to "click here".